library/include/updater.h

```diff
@@ -56,13 +56,14 @@ bool shorebird_init(const struct AppParameters *c_params,
                     const char *c_yaml);
 
 /**
- * Return the active patch number, or NULL if there is no active patch.
+ * The patch number that will boot on the next run of the app, or 0 if there is
+ * no next patch.
  */
-SHOREBIRD_EXPORT char *shorebird_next_boot_patch_number(void);
+SHOREBIRD_EXPORT uintptr_t shorebird_next_boot_patch_number(void);
```

Comment:
You want usize here, I think?

Reply:
This is a C header, so I'm not sure `usize` exists. `usize` is "The pointer-sized unsigned integer type", so I think `uintptr_t` is the correct C-land representation of the bits.

Reply:
Oh yeah, sorry.  I *think* that's size_t then?  Which is in stddef.h?

Reply:
My data is https://locka99.gitbooks.io/a-guide-to-porting-c-to-rust/content/features_of_rust/types.html 🤷

Reply:
I saw that and https://stackoverflow.com/a/1464559 as well. fwiw, this header is autogenerated

Reply:
https://stackoverflow.com/questions/1464174/size-t-vs-uintptr-t

This stuff is all terrible. :)  Hence why we just write in higher level languages.

Reply:
> I saw that and https://stackoverflow.com/a/1464559 as well. fwiw, this header is autogenerated by cbindgen.

Oh.  Fine then.  I definitely trust the autogen more than my googling!  Thank you!